internal/net/message_manager.go

```diff
@@ -743,8 +747,12 @@ func (ms *peerMessageSender) handleRequestWrite(metaMessage MessageInfo) {
 			}
 		}
 		if ms.singleMes > streamReuseTries {
-			_ = ms.s.Close()
-			ms.s = nil
+			// don't close nil stream,
+			// a new stream will be created in the next iteration by ms.prep()
+			if ms.s != nil {
+				_ = ms.s.Close()
```

Comment:
Please consider and log the error does'nt cost a lot

Reply:
OK, you're right, I'll do it in a new PR